### PR TITLE
Added new terms

### PR DESCRIPTION
**Pull request by celyes**
added the following new terms:
* asynchronous
* blocking call
* clone
* construct
* current working directory
* detach
* spawn
* standard input
* standard output
* synchronous
* thread

### CODE REVIEW

File: i18n.json [hunk 1]
```diff
@@ -165,14 +177,19 @@
...
+  { "term": "SPAWN", "wAR": ["انتاج", "توليد"], "wEN": "spawn" },
   { "term": "STABLE", "wAR": "مستقر", "wEN": "stable" },
   { "term": "STATE", "wAR": "الحالة", "wEN": "state" },
   { "term": "STATEMENT", "wAR": "بيان", "wEN": "statment" },
   { "term": "STATUS", "wAR": ["حالة", "وضع"], "wEN": "status" },
+  { "term": "STDIN", "wAR": ["المدخلات القياسية"], "wEN": "standard input" },
+  { "term": "STDOUT", "wAR": ["المخرجات القياسية"], "wEN": "standard output" },
   { "term": "STRICT_MODE", "wAR": "الوضع الصارم", "wEN": "strict mode" },
   { "term": "STRUCTURE", "wAR": ["بنية", "هيكلية"], "wEN": "structure" },
+  { "term": "SYNCHRONOUS", "wAR": ["غير متزامن", "غير متواقت"], "wEN": "synchronous" },
```
Comment (Aissaoui-Ahmed) [inline i18n.json]:
```suggestion
  { "term": "SYNCHRONOUS", "wAR": ["متزامن", "متواقت"], "wEN": "synchronous" },
```

File: i18n.json [hunk 2]
```diff
@@ -165,14 +177,19 @@
...
   { "term": "SYNTAX", "wAR": "صيغة", "wEN": "syntax" },
   { "term": "TAG", "wAR": "وسم", "wEN": "tag" },
+  { "term": "THREAD", "wAR": "خيط", "wEN": "thread" },
```
Comment (Aissaoui-Ahmed) [inline i18n.json]:
```suggestion
  { "term": "THREAD", "wAR": "برهة معالجة", "wEN": "thread" },
```

File: i18n.json [hunk 3]
```diff
@@ -33,33 +31,46 @@
   { "term": "AUTHORIZATION", "wAR": "ترخيص", "wEN": "authorization" },
   { "term": "BASIC", "wAR": "أساسي", "wEN": "basic" },
   { "term": "BIND", "wAR": "ربط", "wEN": "bind" },
+  { 
+    "term": "BLOCKING_CALL",
```
Comment (Aissaoui-Ahmed) [inline i18n.json]:
```suggestion
    "term": "BLOCKING", 
```

File: i18n.json [hunk 4]
```diff
@@ -33,33 +31,46 @@
   { "term": "AUTHORIZATION", "wAR": "ترخيص", "wEN": "authorization" },
   { "term": "BASIC", "wAR": "أساسي", "wEN": "basic" },
   { "term": "BIND", "wAR": "ربط", "wEN": "bind" },
+  { 
+    "term": "BLOCKING_CALL", 
+    "wAR": ["استدعاء مؤجل", "استدعاء محجوب", "استدعاء غير متزامن"],
```
Comment (Aissaoui-Ahmed) [inline i18n.json]:
```suggestion
    "wAR": ["إنتظار الحدث", "استدعاء متزامن"], 
```

File: i18n.json [hunk 5]
```diff
@@ -33,33 +31,46 @@
   { "term": "AUTHORIZATION", "wAR": "ترخيص", "wEN": "authorization" },
   { "term": "BASIC", "wAR": "أساسي", "wEN": "basic" },
   { "term": "BIND", "wAR": "ربط", "wEN": "bind" },
+  { 
+    "term": "BLOCKING_CALL", 
+    "wAR": ["استدعاء مؤجل", "استدعاء محجوب", "استدعاء غير متزامن"], 
+    "wEN": "blocking call"
```
Comment (Aissaoui-Ahmed) [inline i18n.json]:
```suggestion
    "wEN": "blocking" 
```

**Comment by Aissaoui-Ahmed** [comment]
PLZ squash

File: i18n.json [hunk 6]
```diff
@@ -22,65 +22,79 @@
...
-    "term": "AUTHORIZATION",
-    "wAR": "ترخيص",
-    "wEN": "authorization"
-  },
+  { "term": "AUTHORIZATION", "wAR": "ترخيص", "wEN": "authorization" },
   { "term": "BASIC", "wAR": "أساسي", "wEN": "basic" },
   { "term": "BIND", "wAR": "ربط", "wEN": "bind" },
+  { 
+    "term": "BLOCKING_CALL", 
+    "wAR": ["استدعاء مؤجل", "استدعاء محجوب", "استدعاء غير متزامن"],
```
Comment (3imed-jaberi) [inline i18n.json]:
```suggestion
    "wAR": ["استدعاء متزامن"], 
```

Comment (Aissaoui-Ahmed) [reply i18n.json]: [same :white_check_mark: ](https://github.com/softvenue/i18n/pull/36#discussion_r313170028)

Comment (3imed-jaberi) [reply i18n.json]: ( "wAR": ["استدعاء متزامن"] ) !==  ( "wAR": ["إنتظار الحدث", "استدعاء متزامن"] ) 
@Aissaoui-Ahmed 😄 !

File: i18n.json (hunk 1 above)
Comment (celyes) [reply i18n.json]: indeed you are correct.

File: i18n.json (hunk 2 above)
Comment (celyes) [reply i18n.json]: i checked the term and it's correct... thread = خيط

File: i18n.json (hunk 4 above)
Comment (celyes) [reply i18n.json]: انتظار الحدث has no context in here

File: i18n.json (hunk 5 above)
Comment (celyes) [reply i18n.json]: leave it as it is for the context purpose

File: i18n.json (hunk 2 above)
Comment (Aissaoui-Ahmed) [reply i18n.json]: See [Thread ](https://simple.wikipedia.org/wiki/Thread_(computer_science))!

File: i18n.json (hunk 4 above)
Comment (Aissaoui-Ahmed) [reply i18n.json]: [BLOCKING](https://stackoverflow.com/a/2407601/5998913) ❔

**Pull request merged**